### ISSUE TITLE
Training spec #2

### ISSUE THREAD

**Issue by jisngprk** [affjljoo3581/GPT2#7]
Could you share other details of the training results in the comment of the issue which has loss of **3.2398** ? 
- https://github.com/affjljoo3581/GPT2/issues/6#issuecomment-791893107

For example, the things such as scheduler, optimizer beta1, beta2, dropout probability, gradient clipping, learning rate, warmup step, layer normalization, etc.

I just know about some training tips for parameter configuration.

Thank you.

**Comment by jisngprk** [comment]
Thank you!
